filters/ThirdParty/filter_104_EasyListChina/template.txt

```diff
@@ -1,6 +1,9 @@
 !
 @include "https://easylist-downloads.adblockplus.org/easylistchina.txt" /exclude="../../exclusions.txt"
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/47732
+!+ PLATFORM(ios, ext_safari)
+@@$script,subdocument,third-party,websocket,xmlhttprequest,domain=h5.17k.com
```

Comment:
does it exclude the necessary requests(which are blocked by Easylist China)?

Reply:
It works fine on my iPad as mentioned (even without that rule while EasyList China is active).

Reply:
can user check it? 
That rule must block something. You can check it in Safari console with remote debugging.